pkg/minikube/cluster/cluster.go

```diff
@@ -646,3 +671,57 @@ func IsMinikubeRunning(api libmachine.API) bool {
 	}
 	return true
 }
+
+// isMachineExists checks if virtual machine does not exist
+// if the virtual machine exists, return true
+func isMachineExists(vmDriver string, s state.State, err error) (bool, error) {
```

Comment:
english nit: change this to, MachineExists , or DoesMachineExists. (shorter better)

Reply:
renamed to machineExists func